common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -538,8 +539,8 @@ def test_full_pipeline_succeeds_for_linking_account(self):
         actions.do_complete(request.backend, social_views._do_login,  # pylint: disable=protected-access
                             request=request)
 
-        student_views.signin_user(strategy.request)
-        student_views.login_user(strategy.request)
+        signin_user(strategy.request)
```

Comment:
Food for thought/Nit: If we imported `deprecated` and used `deprecated.signin_user`, would it make it more clear throughout what code is using deprecated methods, and in-turn is either deprecated or should be updated to use a new pattern? NOTE: If you do this, there are some other files as well.  If not, no worries.

Reply:
Good idea.  Let's do in next iteration.